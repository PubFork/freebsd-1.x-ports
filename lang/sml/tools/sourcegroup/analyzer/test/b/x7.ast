- Parse.parseSource "b/x7.sml";
[closing b/x7.sml]

[Major collection... 59% used (2437408/4129896), 1890 msec]
val it =
  SeqDec
    [MarkDec
       (SeqDec
          [MarkDec
             (StrDec
                [MarkStrb
                   (Strb
                      {constraint=NONE,
                       def=MarkStr
                             (StructStr
                                (SeqDec
                                   [MarkDec
                                      (ValDec
                                         [MarkVb
                                            (Vb
                                               {exp=MarkExp (IntExp #,29,30),
                                                pat=VarPat [prim?]},27,30)],23,
                                       30)]),16,34),name=prim?},12,34)],2,34),
           MarkDec
             (StrDec
                [MarkStrb
                   (Strb
                      {constraint=NONE,
                       def=MarkStr
                             (StructStr
                                (SeqDec
                                   [MarkDec
                                      (ValDec
                                         [MarkVb
                                            (Vb
                                               {exp=MarkExp (IntExp #,62,63),
                                                pat=VarPat [prim?]},60,63)],56,
                                       63),
                                    MarkDec
                                      (ValDec
                                         [MarkVb
                                            (Vb
                                               {exp=MarkExp (VarExp #,72,75),
                                                pat=VarPat [prim?]},68,75)],64,
                                       75)]),49,79),name=prim?},45,79)],35,
              79)],2,79)] : System.Compile.ast
- 