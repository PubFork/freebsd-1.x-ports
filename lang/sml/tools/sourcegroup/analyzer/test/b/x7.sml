structure A = struct val x=4 end
structure B = struct val y=5 val b = A.x end
