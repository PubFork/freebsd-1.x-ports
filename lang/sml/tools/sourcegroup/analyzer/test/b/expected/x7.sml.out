source sml "b/x7.sml"
  export structure A B ;

